armory/art_experimental/defences/mp3_compression_channelized.py

```diff
@@ -25,7 +25,9 @@ def __call__(self, x, y=None):
             )
 
         # add a channel axis
-        x = np.expand_dims(x, axis=-1)
+        for i in range(len(x)):
+            x[i] = np.expand_dims(x[i], axis=-1)
```

Comment:
Can we assign this to another variable so that the input array is not overwritten?  Also, can the `axis` parameter be set based on whether `channels_first` is True or False? Otherwise, we should assert that `channels_first` is False

Reply:
@kevinmerchant do you want to go ahead and make the changes as needed to be backward compatible and future proof?

As for WER increasing in defended model, this made sense because the DFT attack is not additive (only subtractive), and compression shouldn't improve performance.

Reply:
Actually, as of ART 1.4.2, this art_experimental is no longer needed.

Reply:
Deleted this file in the updated submission